server/index.js

```diff
@@ -19,44 +20,12 @@ app.use(function(req, res, next) {
   next();
 });
 
-app.get('/currentSong/:songid', (req, res) => {
-  console.log('being hit with GET req for current song info');
-  db.getCurrentSong(req, res);
+app.get('/:songid', (req, res) => {
```

Comment:
should this be different than the endpoint you're serving static files to? not sure if this will work once you're trying to get your bundled file from your proxy